packages/wrangler/src/index.ts

```diff
@@ -431,6 +432,15 @@ export function createCLIParser(argv: string[]) {
 		return d1(d1Yargs.command(subHelp));
 	});
 
+	// ai
+	wrangler.command(
+		"ai",
+		"🤖  Interact with Constellation AI models",
+		(aiYargs) => {
+			return ai(aiYargs.command(subHelp));
+		}
+	);
+
```

Comment:
`wrangler ai` is a really general command name. Usually Wrangler commands refer to the brand name for the service (i.e. `wrangler r2` rather than `wrangler object`. What about `wrangler constellation`?